### PR TITLE
add gitignore explicitly

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

File: lib/utils/copy-dir.js [hunk 1]
```diff
@@ -14,10 +14,15 @@ module.exports = function copyDir(opts) {
...
   return new Promise(function(resolve, reject) {
-    const stopCopySpinner = output.wait('Copying files');
+    const stopCopySpinner = output.wait("Copying files");
 
-    fs
-      .copy(templatePath, projectPath)
+    fs.copy(templatePath, projectPath)
+      .then(function() {
+        return fs.move(
+          path.resolve(projectPath, "./.gitignore"),
```
Comment (sneridagh) [inline lib/utils/copy-dir.js]: Moving the same file into the same file?

Comment (sneridagh) [reply lib/utils/copy-dir.js]: When it's packaged, the .gitignore will be gone, so you have to have it (in volto-starter-kit) as gitignore (as the CRA guys does), then move it once the folder is copied over

**Review by sneridagh — CHANGES_REQUESTED** [review]
See comments

**Comment by nileshgulia1** [comment]
@sneridagh It works now.

**Comment by sneridagh** [comment]
This fixed #5

**Pull request merged**